team-policy.md

```diff
@@ -0,0 +1,144 @@
+* Reports wider context to the team
+
+### Facilitator
+
+* Current Holder: [@yaahc](https://github.com/yaahc)
+* Next Feedback Date: Wednesday 2023-02-01
+
+In order to be present with each other, we need a good facilitator.  The facilitator runs meetings according to the format of meetings and decision making adopted by the group. Leader and facilitator are separate roles because facilitation and overseeing operations are different skill sets. They can be held by the same individual.
+
+Responsibilities:
+
+* Facilitates team meetings.
+* Pays attention to equivalence[^5] during meetings.
+* Supports planning of the agenda.
+* Distinguishes facilitator voice from team member voice.
+
+### Scribe
+
+* Current Holder: [@calebcartwright](https://github.com/calebcartwright)
+* Next Feedback Date: TODO
+
+In order to manage continuity with the team’s past, we need to have written records. The scribe manages the notes during the meeting, makes sure the minutes are distributed and accessible.
+
+Responsibilities:
+
+* Makes sure meeting minutes are taken, approved, and stored.
+* Keeps track of all documents of the team.
+* Default interpreter of meeting minutes in cases of disagreement.
+* Supports planning the agenda from the backlog.
+    * Tracking when policies and roles are due for review.
+
+## Meetings
+
+The style team meets weekly on Wednesdays at 12:30pm PST [(everytimezone link)](https://everytimezone.com/s/3f88a253) for a weekly video sync. Agendas are posted in the #T-style zulip stream. 
+
+Unless otherwise noted, all of our meetings are public and open for anyone to attend. You will find the timing and event details on our [style team calendar](https://calendar.google.com/calendar/embed?src=d0564ed914a41cf4915bd5ebe6e2e4ec0ee1293fdc1d09d6f5bdb27d4f91c083%40group.calendar.google.com&ctz=America%2FLos_Angeles). We publish notes and minutes in written form in this github repository.
+
+### Agenda, Backlog, and Minutes
+
+The style team stores its agendas, backlog, and meeting minutes in a single live hackmd document (rotating as necessary when the documents reach their length limit).
+
+The agenda should be prepared in advanced of each meeting by the facilitator with the assistance of the scribe and the lead. The lead is responsible for looking forward and adding agenda items such as new requests from other teams and users or new priorities and goals. The scribe is responsible for looking backwards and adding agenda items from existing work such as items from the backlog or policies that are due for review. The agenda proposal is then presented at the beginning of each meeting by the facilitator for the rest of the team to consent or object too.
+
+Consent not needed to add to items to the backlog, anybody is welcome to add items to the backlog, but consent needed to move from backlog to agenda (since agendas themselves require consent). An item being present on the backlog does not represent a commitment by the style team.
+
```

Comment:
```suggestion
#### Interrupting the Agenda

The style team acknowledges that urgent needs may arise during the meeting, and in order to address these needs, participants may interrupt the meeting by speaking up and immediately saying "Point of Order." This signals to the meeting facilitator and other participants that the following point is not an out of order interruption, but rather an urgent need that must take priority. This tool is particularly intended to allow the scribe to interrupt the meeting when they have unmet needs in their role as minutes taker, but it is available for any team member to use when they feel they have the need.

The meeting facilitator will then pause the current discussion and allow the participant with the point of order to state their concern. After the urgent matter is resolved, the facilitator will resume the previous discussion or return to the agenda. This policy is intended to ensure that the team can address urgent issues in a timely manner without interrupting the flow of the meeting unnecessarily.

```

Reply:
> This tool is particularly intended to allow the scribe to interrupt the meeting when they have unmet needs in their role as minutes taker, but it is available for any team member to use when they feel it is necessary.

I would change this to:

"This tool is designed for meta-level concerns that take priority over the current discussion, such as the scribe interrupting the meeting to address issues preventing the recording of minutes (and stopping the discussion until they can resume recording minutes), or a team member interrupting a discussion to raise questions of time bounds or team scope."

> The meeting facilitator will then pause the current discussion and allow the participant with the point of order to state their concern.

"The meeting facilitator will then pause the current discussion, allow the participant raising the point of order to state their concern, and allow the team to address that point of order."